Sources/PerfectMySQL/PerfectMySQL.swift

```diff
@@ -102,6 +102,7 @@ public enum MySQLOpt {
 	//MYSQL_OPT_SSL_VERIFY_SERVER_CERT,
  	MYSQL_PLUGIN_DIR, MYSQL_DEFAULT_AUTH,
 	MYSQL_OPT_BIND,
+    MYSQL_OPT_SSL_MODE,
```

Comment:
Not sure why the diff is showing this miss-aligned. I've doubled checked the resolved PR and everything is aligned properly.